lib/bolt/inventory.rb

```diff
@@ -73,31 +74,56 @@ def get_targets(targets)
       targets.map { |t| update_target(t) }
     end
 
-    # Should this be a public method?
     def config_for(node_name)
       data = @groups.data_for(node_name)
       if data
         Bolt::Util.symbolize_keys(data['config'])
       end
     end
 
+    def set_var(target, key, value)
+      # TODO: If target_vars is frozen, how do we set values from plans?
```

Comment:
The idea isn't to freeze target_vars but to freeze the hash for each targets. When we want to change a value we merge and reassign.
```ruby
host = target.host
@target_vars[host] = @target_vars[host].merge(key => value).freeze
nil
```

Reply:
Ah, I see